omnistrate_tests/test_operator_failover_cluster.py

```
@@ -0,0 +1,357 @@
+            nodeInstanceType=args.instance_type,
+            enableTLS=args.tls,
+            RDBPersistenceConfig=args.rdb_config,
+            AOFPersistenceConfig=args.aof_config,
+            hostCount=args.host_count,
+            clusterReplicas=args.cluster_replicas,
+            enableDebugCommand=args.debug_command,
+            adminPassword=password,
+            custom_network_id=network.network_id if network else None,
+
+        )
+        
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)